### PR TITLE
Batching

### PR DESCRIPTION
**Pull request by geraintpalmer**
Implementing batching distributions for more than one customer to arrive simultaneously.

### CODE REVIEW

**Comment by geraintpalmer** [comment]
How does this look @drvinceknight would like your opinion?

**Comment by drvinceknight** [comment]
Just back today Ger, will take a look!

File: docs/Guides/batching.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    >>> import ciw
+    >>> N = ciw.create_network(
+    ...     Arrival_distributions=[['Deterministic', 18.5]],
+    ...     Service_distributions=[['Deterministic', 3.0]],
+    ...     Batching_distributions=[['Deterministic', 3]],
+    ...     Number_of_servers=[1]
+    ... )
+
+If this system is simulated for 30 time units, only one arrival event will occur.
+However, 3 customers will arrival that that node simultaneously.
```
Comment (drvinceknight) [inline docs/Guides/batching.rst]: typo

File: docs/Guides/batching.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Note that, just like arrival and service distributions, batching distributions can be defined for multiple nodes and multiple customer classes, using lists and dictionaries::
+
+    Batching_distributions={
+        'Class 0': [['Deterministic', 3],
+                    ['Deterministic', 1]],
+        'Class 1': [['Deterministic', 2],
+                    ['Deterministic', 2]]},
+
+Note:
```
Comment (drvinceknight) [inline docs/Guides/batching.rst]: You say the word `Note` a lot.

**Pull request merged**